build/gulpfile.js

```diff
@@ -56,12 +58,17 @@ module.exports = function(gulp, config, pluginName){
         module: 'commonjs',
         declarationFiles: true,
         noExternalResolve: false,
-        removeComments: true
-      })))
-      .on('error', plugins.notify.onError({
```

Comment:
this serves as a notifier: https://github.com/hawkular/hawkular-ui-components/pull/74/files#diff-b20c7230de4da834e815c06534cab810R63